### ISSUE TITLE
Position of path

### ISSUE THREAD

**Issue by D2Designs** [JoelBesada/scrollpath#24]
Is there a way to change the position of the path so that it is not relative to the center of the browser window? I would like to incorporate this into a DIV on a page with other elements, and only want the content within the DIV to scroll along the path, but with the path positioning being relative to the center of the DIV not the browser window. Is this possible?

**Comment by pankajthakur007** [comment]
yes it is possible...  set your div according to you change in style.css and demo.js . in demo.js set MoveTo()  like .moveTo(680, 340, {name: "start"})   try it,,,

**Comment by magus824** [comment]
I was wondering if you could elaborate more on how to center this within a div? For example, I currently have a div that is a width of 600, a height of 600, and is position:relative. When I put the scrollpath area inside of this div, it still arranges itself so that it's center is the center of the browser window, and not at the middle of the div surrounding it (which would be 300, 300). Can you help me understand how to set up Scrollpath so that it centers itself within a div, and not to the center of the browser?
